src/lib/backend.ts

```diff
@@ -339,14 +339,23 @@ export class ElectronBackend implements Backend {
 
   /** Activates the Node SDK for the main process. */
   private async installMainHandler(): Promise<boolean> {
+    const options = this.frontend.getOptions();
+    // tslint:disable-next-line:no-unbound-method
+    const customHandler = options.onFatalError;
```

Comment:
@jan-auer is there a way to do this without disabling `no-unbound-method`?

Reply:
You could trick the linter for now by writing:
```ts
    const { onFatalError } = this.frontend.getOptions();
```